docs/pages/includes/database-access/aws-troubleshooting.mdx

```diff
@@ -0,0 +1,28 @@
+### Certificate Error
+
+If your `tsh db connect` error includes the following text you likely have a RDS database created before July 28, 2020.
+
+```code
+x509: certificate relies on legacy Common Name field, use SANs instead
+```
+
+AWS provides instructions to rotate your [SSL/TLS certificate](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html). Even if
+your database is using the `rda-ca-2019` certificate and was created before July 28, 2020 that 
+will require rotation.
+
+### No Credential Providers Error
+
+If you see the error ` NoCredentialProviders: no valid providers in chain` in logs then Teleport
+is not detecting the required Credentials to connect via AWS IAM permissions.  Check
+the credentials or security role has been applied in the machine running the Teleport database service.
+
+### Timeout Errors
+
+The Teleport database service needs connectivity to the database endpoints.  That may require
+enabling inbound traffic within the VPC or routing rules. Using the `nc`
```

Comment:
The VPC where your database servers are running?

Reply:
@ptgott modified language to make more clear